src/components/giving-tuesday/Countdown.js

```diff
@@ -50,71 +58,44 @@ const Countdown = ({ blok }) => {
 
   return (
     <SbEditable content={blok}>
-      {days + hours + minutes + seconds <= 0 ? (
-        <div
-          className={classNames('countdown-wrapper', {
-            ['has-days']: days > 0,
-          })}
-        >
-          {days > 0 && hasDays && (
-            <CountdownPie className="pie-days" descriptor="days" percent={0}>
-              0
-            </CountdownPie>
-          )}
-          <CountdownPie clasName="pie-hours" descriptor="hours" percent={0}>
-            0
-          </CountdownPie>
-          <CountdownPie clasName="pie-minutes" descriptor="minutes" percent={0}>
-            0
-          </CountdownPie>
+      <div
+        aria-atomic="true"
+        className={classNames('countdown-wrapper', {
+          ['has-days']: days > 0 && hasDays,
+        })}
+        role="timer"
+      >
+        {days > 0 && hasDays && (
           <CountdownPie
-            className="pie-seconds"
-            descriptor="seconds"
-            percent={0}
+            className={daysClassName}
+            descriptor={getDescriptorString('day', days)}
+            percent={noTime ? 0 : (days / 29) * 100}
```

Comment:
29 days?

Reply:
good catch @sherakama, i believe that was hardcoded for the specific countdown in place. i will rework that.